script/Ciel/Day 13/13_01_02_CIEL13_7C.txt

```diff
@@ -206,27 +206,27 @@ The black keys that rained from the sky annihilates the ring of the Dead without
 [sha:f1e6072ca57fe9dcc1b3a57036da19273188fcca]{
 -- Page 293, Offset 36137.
 -- ノエルは殺意に満ちた目で俺を―――すべての手勢が消え去った地上を見つめ、切り捨てた。
-With murderous eyes―――Noel stares at me and the troops of her Dead being completely wiped clean.
+With murderous eyes, Noel stares at me and her now razed troops, electing to abandon us.
```

Comment:
With eyes full of murderous intent, Noel stares at me, and at her razed troops, electing to abandon us.